src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/RemoveRepresentationPreparerTest.java

```diff
@@ -128,7 +128,7 @@ void it_can_handle_callback() {
 
                 boolean canHandle = removeRepresentationPreparer.canHandle(callbackStage, callback);
 
-                if (event == Event.REMOVE_REPRESENTATION
+                if ((event == Event.REMOVE_REPRESENTATION || event == Event.REMOVE_LEGAL_REPRESENTATIVE)
                     && callbackStage == PreSubmitCallbackStage.ABOUT_TO_START) {
```

Comment:
Test should_write_to_change_organisation_request_field()
 handling_should_throw_if_cannot_actually_handle() 
 should_respond_with_error_when_organisation_policy_not_present()
should include for event REMOVE_LEGAL_REPRESENTATIVE
Suggest using @ParameterizedTest for event

Reply:
Done for the relevant tests